test/UnitTest/v2/test_casesNGSIv2.py

```diff
@@ -33,149 +34,153 @@ def test_getAllSubscription():
 '''
 
 def test_getSubscriptionById():
-	url=brokerIp+"/v2/subscriptions"
-	headers= {'Content-Type': 'application/json'}
-	r=requests.post(url,data=json.dumps(v2data.subscription_data),headers=headers)
-	resp_content=r.content
-	resInJson= resp_content.decode('utf8').replace("'", '"')
-	resp=json.loads(resInJson)
-	resp=resp['subscribeResponse']
-	sid=resp['subscriptionId']
-	get_url=brokerIp+"/v2/subscription/"
-	url=get_url+sid
-	r=requests.get(url,headers=headers)
-	assert r.status_code == 200
-	print("Get subscription by Id testcase passed")
+        url=brokerIp+"/v2/subscriptions"
+        headers= {'Content-Type': 'application/json'}
+        r=requests.post(url,data=json.dumps(v2data.subscription_data),headers=headers)
+        resp_content=r.content
+        resInJson= resp_content.decode('utf8').replace("'", '"')
+        resp=json.loads(resInJson)
+        resp=resp['subscribeResponse']
+        sid=resp['subscriptionId']
+        get_url=brokerIp+"/v2/subscription/"
+        url=get_url+sid
+        r=requests.get(url,headers=headers)
+        assert r.status_code == 200
+        print("Get subscription by Id testcase passed")
+
 
 #testCase 3
 '''
   Testing get subscription by nil id
 '''
 
-def test_getSubscriptionByNilId():	
-	get_url=brokerIp+"/v2/subscription/nil"
-	headers= {'Content-Type': 'application/json'}
-	r=requests.get(get_url,headers=headers)
-	assert r.status_code == 404	
+def test_getSubscriptionByNilId():
+        get_url=brokerIp+"/v2/subscription/nil"
+        headers= {'Content-Type': 'application/json'}
+        r=requests.get(get_url,headers=headers)
+        assert r.status_code == 404
 
 # Test Delete subscription id subscriptionId is persent in the broker
 #testCase 4
 
 '''
-  Test delete subscription by Id 
+  Test delete subscription by Id
 '''
 
 def test_deleteSubscriptionById():
-	url=brokerIp+"/v2/subscriptions"
-	headers= {'Content-Type': 'application/json'}
-	r=requests.post(url,data=json.dumps(v2data.subscription_data),headers=headers)
-	resp_content=r.content
-	resInJson= resp_content.decode('utf8').replace("'", '"')
-	resp=json.loads(resInJson)
-	resp=resp['subscribeResponse']
-	sid=resp['subscriptionId']
-	get_url=brokerIp+"/v2/subscription/"
-	url=get_url+sid
-	r=requests.delete(url,headers=headers)
-	assert r.status_code == 200
+        url=brokerIp+"/v2/subscriptions"
+        headers= {'Content-Type': 'application/json'}
+        r=requests.post(url,data=json.dumps(v2data.subscription_data),headers=headers)
+        resp_content=r.content
+        resInJson= resp_content.decode('utf8').replace("'", '"')
+        resp=json.loads(resInJson)
+        resp=resp['subscribeResponse']
+        sid=resp['subscriptionId']
+        get_url=brokerIp+"/v2/subscription/"
+        url=get_url+sid
+        r=requests.delete(url,headers=headers)
+        assert r.status_code == 200
+
+
 
 #testCase 5
-''' 
+'''
 Test if subscriptionId is not persent in the broker
```

Comment:
Title correction: "Test if subscriptionId is not persent in the broker" => "Test deletion if subscription is not present on Broker"